tools/who_what_benchmark/whowhatbench/model_loaders.py

```diff
@@ -194,6 +194,9 @@ def load_text2image_model(
         logger.info("Using OpenVINO GenAI API")
         model = load_text2image_genai_pipeline(model_id, device, ov_config, **kwargs)
     elif use_hf:
+        if 'empty_adapters' in kwargs:
```

Comment:
please, move it to check_args() https://github.com/openvinotoolkit/openvino.genai/blob/b9a5efdca9b317e88b1be3e4c3e4b56813019f2b/tools/who_what_benchmark/whowhatbench/wwb.py#L200